templates/setup.py.jj2

```diff
@@ -113,7 +113,19 @@ if python_implementation == "PyPy":
 INSTALL_REQUIRES = [
 {% for dependency in dependencies: %}
   {% if ';' not in dependency and not dependency.startswith('#'): %}
+    {% if '#egg=' in dependency: %}
+      {% set dependency = dependency.split('#egg=') %}
+      {% set repo_link, egg_name = dependency[0], dependency[1] %}
+      {% if '#' in egg_name: %}
+        {% set egg_name = egg_name.split('#')[0] %}
+      {% endif %}
+    '{{[repo_link, egg_name] | join('#egg=')}}',
```

Comment:
note jinja bear expects whitespace inside the `{{ .. }}`.  im not sure of the preference in this repo, if any, about that.

Reply:
In this repo, I think without whitespace is used almost exclusively. [ref](https://github.com/moremoban/pypi-mobans/pull/53/files#diff-b400d1f6d4ac5b0a6a40fc0fb30ecf48R30)

Reply:
to be honest, there has not been a style for it. I will create an issue to comply the {{ .. }} .

Reply:
@chfw We may follow [this](https://github.com/coala/bear-docs/blob/master/docs/Jinja2Bear.rst).

Reply:
Or maybe [these rules](https://github.com/coala/bear-docs/blob/master/docs/Jinja2Bear.rst).

Reply:
I like coala jinja2 bear :)

Reply:
@ayan-b , do you know to set coala bar up? it will be useful for pypi-mobans

Reply:
Yup, I can do that.